website/layouts/_shortcodes/step.html

```
@@ -0,0 +1,29 @@
+<li>
+  {{- $rendered := .Inner | page.RenderString -}}
+  {{- /* Skip heading collection when nested inside a tab — tabs.html scans the
+       rendered tab content and collects all headings (including those from steps)
+       after dedup. Only collect for standalone steps (steps not inside a tab). */ -}}
+  {{- $inTab := false -}}
+  {{- with .Parent -}}{{- with .Parent -}}{{- if eq .Name "tab" -}}{{- $inTab = true -}}{{- end -}}{{- end -}}{{- end -}}
+  {{- if not $inTab -}}
+    {{- range findRE `<h[2-6][^>]*id="[^"]+"` $rendered -}}
+      {{- $level := replaceRE `<h([2-6]).*` "$1" . | int -}}
+      {{- $id := replaceRE `.*\bid="([^"]+)".*` "$1" . -}}
+      {{- $textMatches := findRE (printf `<h[2-6][^>]*id="%s"[^>]*>[^<]+` $id) $rendered -}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)